media-libs/libheif/libheif-1.9.1.ebuild

```diff
@@ -6,18 +6,18 @@ EAPI=7
 inherit autotools xdg multilib-minimal
 
 if [[ ${PV} == "9999" ]] ; then
-	EGIT_REPO_URI="https://github.com/strukturag/${PN}.git"
+	EGIT_REPO_URI="https://github.com/strukturag/libheif.git"
 	inherit git-r3
 else
-	SRC_URI="https://github.com/strukturag/${PN}/releases/download/v${PV}/${P}.tar.gz"
+	SRC_URI="https://github.com/strukturag/libheif/releases/download/v${PV}/${P}.tar.gz"
 	KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86"
 fi
 
 DESCRIPTION="ISO/IEC 23008-12:2017 HEIF file format decoder and encoder"
 HOMEPAGE="https://github.com/strukturag/libheif"
 
 LICENSE="GPL-3"
-SLOT="0/1.8"
+SLOT="0/1.9.1"
```

Comment:
Maybe `1.9` is enough since rarely minor versions change ABI. Are you sure this isn't compatible with previous 1.8 either?

Reply:
I don't think it's compatible, but 1.9 will be enough

Reply:
Note that this is about ABI compatibility, not API.

i.e. Will a linked binary against the old version still work correctly against the new one? Were the exposed existing symbols changed or removed? (New ones are fine).

Reply:
Few added symbols, but none removed, so it should be backwards compatible. That means the sub-slot doesn't have to be updated I guess? Previously it was changed on every version bump, so I was working in line with that